### PR TITLE
Change "normalize" to "standardize"

### PR DESCRIPTION
**Pull request by hyanwong**
Is this better @nspope (c.f. https://github.com/tskit-dev/tsdate/pull/232#issuecomment-1381958466 with which I agree)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -8,11 +8,15 @@
   individuals, populations, or sites, aiming to change the tree sequence tables as
   little as possible.
 
+- Not strictly breaking, as not in the published API, but the "normalize" flag
+  in ``get_dates`` and the internal ``normalize`` terminology is changed to
+  ``sandardize`` to better reflect the fact that the maximum (not sum) is one.
```
Comment (nspope) [inline CHANGELOG.rst:13]:
```suggestion
  ``standardize`` to better reflect the fact that the maximum (not sum) is one.
```

**Comment by nspope** [comment]
LGTM!

**Comment by awohns** [comment]
Sounds good!

File: CHANGELOG.rst (hunk 1 above)
Comment (hyanwong) [reply CHANGELOG.rst:13]: Good spot, thanks

**Comment by hyanwong** [comment]
Thanks both

**Pull request merged**